src/system/ThemeModule/AbstractTheme.php

```diff
@@ -52,7 +52,7 @@ public function generateThemedResponse($realm, Response $response)
     {
         $template = $this->config[$realm]['page'];
 
-        return $this->getContainer()->get('templating')->renderResponse($this->name . ':' . $template, array('maincontent' => $response->getContent()));
+        return $this->getContainer()->get('twig')->renderResponse($this->name . ':' . $template, array('maincontent' => $response->getContent()));
```

Comment:
No correction, just a simplification improving performance a bit.
http://symfony.com/blog/new-in-symfony-2-7-twig-as-a-first-class-citizen#using-twig-without-the-templating-sub-system